python/cudf/cudf/_libxx/strings/padding.pyx

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+from cudf._libxx.move cimport move
+from cudf._libxx.cpp.column.column_view cimport column_view
+from cudf._libxx.cpp.scalar.scalar cimport string_scalar
+from cudf._libxx.cpp.types cimport size_type
+from cudf._libxx.column cimport Column
+from cudf._libxx.scalar cimport Scalar
+from libcpp.string cimport string
+
+from cudf._libxx.strings.padding cimport (
+    pad as cpp_pad,
+    zfill as cpp_zfill,
+    pad_side as pad_side
+)
+
+
+def pad(Column source_strings,
+        size_type width,
+        side,
+        fill_char):
+    cdef unique_ptr[column] c_result
+    cdef column_view source_view = source_strings.view()
+
+    cdef pad_side pad_direction
+    cdef string f_char = <string>str(fill_char).encode()
+
+    if side == "left":
+        pad_direction = pad_side.LEFT
+    elif side == "right":
+        pad_direction = pad_side.RIGHT
+    elif side == "both":
+        pad_direction = pad_side.BOTH
```

Comment:
Instead of this lets create a Python Enum similar to what we've done in other Cython bindings and then typecast the enum value to the enum class. See: https://github.com/rapidsai/cudf/blob/branch-0.13/python/cudf/cudf/_libxx/types.pyx